.github/workflows/validate.yml

```diff
@@ -0,0 +1,38 @@
+name: validate
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - 'master'
+    tags:
+      - 'v*'
+  pull_request:
+    branches:
+      - 'master'
+
+env:
+  REPO_SLUG_ORIGIN: "moby/buildkit:master"
```

Comment:
Is this intentionally master?

Reply:
@tonistiigi Same behavior as Travis: https://travis-ci.org/github/docker/buildx/jobs/747558356#L176 but we can change it if you think so.